docs/AboutUs.adoc

```diff
@@ -11,7 +11,7 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 == Project Team
 
 === Simon Lam
-image::Simon.png[width="150", align="left"]
+image::TheElectricBlender.png[width="150", align="left"]
```

Comment:
OOPS. Thanks for fixing the problem huiting!!

Reply:
Good job Hui Ting! Thanks for doing the necessary edits on time.